<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>client_GUI</class>
 <widget class="QMainWindow" name="client_GUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>958</width>
    <height>602</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>150</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0.233831 rgba(91, 133, 170, 255), stop:1 rgba(255, 255, 255, 255))
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="message_view_box">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>871</width>
      <height>361</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background:qlineargradient(spread:pad, x1:0.104, y1:0.142045, x2:1, y2:1, stop:0 rgba(0, 0, 0, 240), stop:1 rgba(198, 228, 234, 255));
border: 2px solid black;
border-radius: 45px;

/*background: QLinearGradient(x1: 0, y1: 0,
                            x2: 1, y2: 1,
                            stop:0 rgba(29,48,87,1),
				stop:0.12 rgba(69,117,94,1),
				stop:0.5 rgba(152,186,236,1),
				stop:0.87 rgba(41,100,124,1),
				stop:1 rgba(74,31,120,1) );
qradialgradient*/</string>
    </property>
    <property name="locale">
     <locale language="Russian" country="Russia"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="message_enter_box">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>440</y>
      <width>581</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="send_button">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>440</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(156, 255, 201);
border: 1px solid white;
border-radius: 45px;</string>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_button">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>480</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 140, 148);
border: 1px solid white;
border-radius: 45px;</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="file_uploader">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>430</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(117, 219, 255);
border: 1px solid white;
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>UploadFile</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="key_enter_box">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>410</y>
      <width>16</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>380</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(255, 243, 255)</string>
    </property>
    <property name="locale">
     <locale language="Russian" country="Russia"/>
    </property>
    <property name="text">
     <string>   CONNECTIONS</string>
    </property>
   </widget>
   <widget class="QLabel" name="clients_count">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>380</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="wake_up_button">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>450</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(117, 219, 255)</string>
    </property>
    <property name="locale">
     <locale language="Russian" country="Kazakhstan"/>
    </property>
    <property name="text">
     <string>Wake Up</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>message_view_box</zorder>
   <zorder>send_button</zorder>
   <zorder>file_uploader</zorder>
   <zorder>clear_button</zorder>
   <zorder>message_enter_box</zorder>
   <zorder>key_enter_box</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>pushButton</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionexit_2">
   <property name="text">
    <string>exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
